src/locators/defaultLocators.js

```diff
@@ -0,0 +1,47 @@
+var clientsidescripts = require('../scripts/clientsidescripts');
```

Comment:
This module tends to grow too much, lets remove it and add the default locators in the profile.conf.js, similar to the reporters.